src/pages/rewards/RewardsPage.tsx

```diff
@@ -1,17 +1,326 @@
 import styled, { AnyStyledComponent } from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { CLIENT_NETWORK_CONFIGS } from '@/constants/networks';
+import { STRING_KEYS } from '@/constants/localization';
+import { ButtonAction, ButtonSize, ButtonType } from '@/constants/buttons';
+import { DialogTypes } from '@/constants/dialogs';
+import { breakpoints } from '@/styles';
+
+import { useAccountBalance, useBreakpoints, useStringGetter } from '@/hooks';
 
 import { layoutMixins } from '@/styles/layoutMixins';
 
+import { Details } from '@/components/Details';
+import { Panel } from '@/components/Panel';
+import { IconName } from '@/components/Icon';
+import { Button } from '@/components/Button';
+import { IconButton } from '@/components/IconButton';
+import { Output, OutputType } from '@/components/Output';
+import { VerticalSeparator } from '@/components/Separator';
+import { WithReceipt } from '@/components/WithReceipt';
+
+import { openDialog } from '@/state/dialogs';
+import { getSelectedNetwork } from '@/state/appSelectors';
+
 import { DYDXBalancePanel } from './DYDXBalancePanel';
+import { Link } from '@/components/Link';
+
+export const RewardsPage = () => {
+  const dispatch = useDispatch();
+  const stringGetter = useStringGetter();
+
+  const selectedNetwork = useSelector(getSelectedNetwork);
+  const { isTablet, isNotTablet } = useBreakpoints();
+
+  const chainId = Number(CLIENT_NETWORK_CONFIGS[selectedNetwork].ethereumChainId);
+
+  const { balance } = useAccountBalance({
+    addressOrDenom: import.meta.env.VITE_V3_TOKEN_ADDRESS,
+    assetSymbol: 'DYDX',
+    chainId,
+    isCosmosChain: false,
+  });
 
-export const RewardsPage = () => (
-  <Styled.Page>
-    <DYDXBalancePanel />
-  </Styled.Page>
-);
+  const tokenBalance = import.meta.env.VITE_V3_TOKEN_ADDRESS ? balance : 0;
+
+  // TODO: replace placeholder URL with real URLs when avaialble
+  const MIGRATE_HELP_URL = 'https://help.dydx.exchange/';
+  const GOVERNANCE_HELP_URL = 'https://help.dydx.exchange/';
+  const STAKING_HELP_URL = 'https://help.dydx.exchange/';
+  const BRIDGE_URL = 'https://bridge.dydx.exchange/';
+
+  return (
+    <Styled.Page>
```

Comment:
can we add back padding for tablet view?

<img width="1182" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/728cf930-8687-4ae3-816d-cbcb51958c32">

Reply:
Weird, there should be padding for tablet view. Maybe that width doesn't hit the table breakpoint yet.

Reply:
ummm i guess it's something with the calculations (clamping etc.) since it's edge case its ok